Digizuite/ServiceHttpWrapper.cs

```diff
@@ -43,6 +45,7 @@ public class ServiceHttpWrapper
             {ServiceType.LoginService, "http://digizuitecore_loginservice"},
             {ServiceType.LegacyService, "http://digizuitecore_legacyservice"},
             {ServiceType.NotificationService, "http://digizuitecore_notificationservice"},
+            {ServiceType.FileService, "/DigizuiteCore/digizuitecore_fileservice"},
```

Comment:
That isn't correct, take a look at the other patterns here ;)

Reply:
Yikes, I much have been drunk :O